server/game/Card.js

```diff
@@ -356,7 +386,8 @@ class Card extends EffectSource {
         if(this.anyEffect('changeHouse')) {
             combinedHouses = combinedHouses.concat((this.getEffects('changeHouse')));
         } else {
-            combinedHouses.push(this.printedHouse);
+            let copyEffect = this.mostRecentEffect('copyCard');
```

Comment:
@jeremylarner : curious, will this cover the fact that Mimic Gel copies a card, but not the house?

Reply:
No, that will need to be dealt with separately